From mingo@elte.hu Tue Jul 24 02:16:44 2007
From: Ingo Molnar <mingo@elte.hu>
Date: Tue, 24 Jul 2007 11:16:37 +0200
Subject: pci/hotplug/cpqphp_ctrl.c: remove stale BKL use
To: Greg KH <gregkh@suse.de>
Cc: linux-kernel@vger.kernel.org
Message-ID: <20070724091637.GA6554@elte.hu>
Content-Disposition: inline


Subject: pci/hotplug/cpqphp_ctrl.c: remove stale BKL use
From: Ingo Molnar <mingo@elte.hu>

remove stale BKL use from drivers/pci/hotplug/cpqphp_ctrl.c.

Signed-off-by: Ingo Molnar <mingo@elte.hu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/hotplug/cpqphp_ctrl.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/drivers/pci/hotplug/cpqphp_ctrl.c
+++ b/drivers/pci/hotplug/cpqphp_ctrl.c
@@ -1746,10 +1746,8 @@ static void pushbutton_helper_thread(uns
 static int event_thread(void* data)
 {
 	struct controller *ctrl;
-	lock_kernel();
+
 	daemonize("phpd_event");
-	
-	unlock_kernel();
 
 	while (1) {
 		dbg("!!!!event_thread sleeping\n");
